ledger/sandbox/src/main/scala/com/digitalasset/platform/sandboxnext/Runner.scala

```diff
@@ -120,43 +121,46 @@ class Runner(config: SandboxConfig) extends ResourceOwner[Port] {
                   config.metricsReporter,
                   config.metricsReportingInterval,
                 )
-                _ <- startupMode match {
-                  case StartupMode.MigrateAndStart =>
-                    ResourceOwner.successful(())
-                  case StartupMode.ResetAndStart =>
-                    // Resetting through Flyway removes all tables in the database schema.
-                    // Therefore we don't need to "reset" the KV Ledger and Index separately.
-                    ResourceOwner.forFuture(() => new FlywayMigrations(indexJdbcUrl).reset())
```

Comment:
I'm surprised we don't need to reset the index. Does something else do this?

Reply:
The reset is not integrated into the startup procedure, i.e. `MigrateAndStart` vs. `ResetAndStart`.